src/firefox/remote.js

```diff
@@ -16,11 +16,13 @@ export const REMOTE_PORT = 6005;
 
 
 // RemoteFirefox types and implementation
-
+/* eslint-disable import/order, import/imports-first, import/no-extraneous-dependencies */
+//This is related to Flow and will be stripped in production hence won't be treated as a dependency at all
 import type FirefoxClient from 'firefox-client';
 
 export type FirefoxConnectorFn =
   (port?: number) => Promise<FirefoxClient>;
+/* eslint-disable import/order, import/imports-first */
```

Comment:
@shubheksha This seems that it should be an `eslint-enable` comment

Also, I'm wondering if `export type FirefoxConnectorFn = ...` needs to be inside the `eslint-disable` comment.